deployment/ccip/changeset/v1_6/config.go

```diff
@@ -0,0 +1,181 @@
+			DeltaRound:                              15 * time.Second,
+			DeltaGrace:                              globals.OcrDeltaGrace,
+			DeltaCertifiedCommitRequest:             globals.OcrDeltaCertifiedCommitRequest,
+			DeltaStage:                              25 * time.Second, // TransmissionDelayMultiplier overrides DeltaStage
+			Rmax:                                    globals.OcrRMax,
+			MaxDurationQuery:                        globals.OcrMaxDurationQuery,
+			MaxDurationObservation:                  13 * time.Second,
+			MaxDurationShouldAcceptAttestedReport:   globals.OcrMaxDurationShouldAcceptAttestedReport,
+			MaxDurationShouldTransmitAcceptedReport: 10 * time.Second,
+		},
+		ExecuteOffChainConfig: &globals.DefaultExecuteOffChainCfg,
+	}
+
+	DefaultOCRParamsForCommitForETH = CCIPOCRParams{
+		OCRParameters: types.OCRParameters{
+			DeltaProgress:                           120 * time.Second,
+			DeltaResend:                             globals.OcrDeltaResend,
+			DeltaInitial:                            globals.OcrDeltaInitial,
+			DeltaRound:                              90 * time.Second,
+			DeltaGrace:                              globals.OcrDeltaGrace,
+			DeltaCertifiedCommitRequest:             globals.OcrDeltaCertifiedCommitRequest,
+			DeltaStage:                              60 * time.Second, // TransmissionDelayMultiplier overrides DeltaStage
+			Rmax:                                    globals.OcrRMax,
+			MaxDurationQuery:                        globals.OcrMaxDurationQuery,
+			MaxDurationObservation:                  35 * time.Second,
+			MaxDurationShouldAcceptAttestedReport:   globals.OcrMaxDurationShouldAcceptAttestedReport,
+			MaxDurationShouldTransmitAcceptedReport: 10 * time.Second,
+		},
+		CommitOffChainConfig: &globals.DefaultCommitOffChainCfg,
+	}
+
+	DefaultOCRParamsForExecForETH = CCIPOCRParams{
+		OCRParameters: types.OCRParameters{
+			DeltaProgress:                           100 * time.Second,
+			DeltaResend:                             globals.OcrDeltaResend,
+			DeltaInitial:                            globals.OcrDeltaInitial,
+			DeltaRound:                              90 * time.Second,
+			DeltaGrace:                              globals.OcrDeltaGrace,
+			DeltaCertifiedCommitRequest:             globals.OcrDeltaCertifiedCommitRequest,
+			DeltaStage:                              60 * time.Second, // TransmissionDelayMultiplier overrides DeltaStage
+			Rmax:                                    globals.OcrRMax,
+			MaxDurationQuery:                        globals.OcrMaxDurationQuery,
+			MaxDurationObservation:                  20 * time.Second,
+			MaxDurationShouldAcceptAttestedReport:   globals.OcrMaxDurationShouldAcceptAttestedReport,
+			MaxDurationShouldTransmitAcceptedReport: 8 * time.Second,
+		},
+		ExecuteOffChainConfig: &globals.DefaultExecuteOffChainCfg,
+	}
+
+	// Used for only testing with simulated chains
+	OcrParamsForTest = CCIPOCRParams{
```

Comment:
Without this I was getting a lot of flaky tests, need to ensure if at all we want real chain values in simulated